backend/fms_core/template_importer/row_handlers/sample_pooling/pool.py

```diff
@@ -26,7 +26,7 @@ def process_row_inner(self, protocol, imported_template, samples_info, pool, poo
                 pool_is_library = False
                 set_kind = set(sample["Source Sample"].derived_samples.first().sample_kind.name for sample in samples_info)
                 # Assumes source sample pools did not allow different individuals to be pooled
-                set_individual = set(sample["Source Sample"].derived_samples.first().biosample.individual.name for sample in samples_info)
+                set_individual = set(sample["Source Sample"].derived_samples.first().biosample.individual for sample in samples_info)
```

Comment:
maybe replace individual by individual _id copying the whole individuals may be a bit overkill for this use.

Reply:
Great idea